kuma/static/styles/components/wiki/communitybox.styl

```diff
@@ -0,0 +1,79 @@
+/*
+  blue box on footer of many landing pages
+  Generated by macros:
+    https://developer.mozilla.org/en-US/docs/Template:TopicBox
+    https://developer.mozilla.org/en-US/docs/Template:CommunityBox
+*/
+$communitybox-spacing = 13px;
+
+.communitybox {
+    padding: $communitybox-spacing;
+    background-color: $homepage-background-color;
+    background-image: url($path-to-images + "blueprint.png");
+    font-size: $base-font-size;
+
+    &,
+    & a,
+    & a:hover,
+    & a:focus,
+    & a:visited {
+        color: #fff;
+    }
+
+    strong {
+        font-size: $base-bump-font-size;
+    }
+
+    ul {
+        padding: 0;
+        margin-bottom: 0;
+        list-style-type: none;
+    }
+
+    h2 {
+        margin-bottom: 0;
+    }
+}
+
+/* subheading */
+.communitysubhead {
+    font-size: $base-bump-font-size;
+    font-weight: bold;
+
+    /* only need margin to balance it with content in other columns */
+    .column-half & {
+        margin-top: $communitybox-spacing;
+        margin-bottom: $content-vertical-spacing;
+    }
+}
+
+/* links to mailing list */
+ul.communitymailinglist { /* needs specificity to get past .text-content ul */
+    margin: $communitybox-spacing 0 ($content-vertical-spacing * -1) 0;
+}
+
+.communitymailinglist li {
+    display: inline-block;
+    margin: 0 0 $content-vertical-spacing 0;
```

Comment:
Just a nitpick since our compressor might do it anyway. We could alternatively use `margin: 0 0 $content-vertical-spacing`.